portal/static/portal/sass/styles.scss

```diff
@@ -429,7 +435,11 @@ button,
   color: $color-text-secondary;
 }
 
-.button--secondary--dark  {
+.button--secondary--dark,
```

Comment:
This actually represents the secondary buttons on white background, their text should stay grey.